inference_client/client.py

```diff
@@ -22,24 +22,33 @@ def __init__(
         """
 
         try:
-            self._auth_token = login(token)
+            self._auth_token = login(token) if token else None
         except Exception:
             raise ValueError(
                 f'Invalid or expired auth token. Please re-enter your token and try again.'
             ) from None
 
     @lru_cache(maxsize=10)
-    def get_model(self, model_name: str):
+    def get_model(self, model_name_or_endpoint: str):
```

Comment:
Can we use separated keywords for this purpose?

Reply:
good idea